clang-tools-extra/clangd/InlayHints.cpp

```diff
@@ -372,6 +381,25 @@ maybeDropCxxExplicitObjectParameters(ArrayRef<const ParmVarDecl *> Params) {
   return Params;
 }
 
+template <typename R, typename P>
+std::string joinAndTruncate(R &&Range, size_t MaxLength,
```

Comment:
Now that the function has only one call site, can we drop the `GetAsStringFunction` parameter and just assume the elements are themselves strings?